### PR TITLE
Update AppVeyor comments in Implicit.Remoting.Tests.ps1

### PR DESCRIPTION
**Pull request by RDIL**
# PR Summary
We are getting close to no AppVeyor references 🎉 
Updates comments as Azure is now used

## PR Context
AppVeyor removed ( #8686 )

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [X] Issue filed:  #8686 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by PaulHigin — APPROVED** [review]

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1 [hunk 1]
```diff
@@ -383,7 +383,7 @@ try
         }
 
         ## It requires 'New-PSSession' to work with implicit credential to allow proxied command to create new session.
-        ## Implicit credential doesn't work in AppVeyor builder, so mark all tests here '-pending'.
+        ## Implicit credential doesn't work in the Azure DevOps builder, so mark all tests here '-pending'.
```
[the remark this answers is not in the document]
Comment (RDIL) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1]: Just started a build to validate this.

**Comment by RDIL** [comment]
@TravisEz13 yeah no it doesn't work

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (RDIL) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1]: Yeah no doesn't work

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1 [hunk 2]
```diff
@@ -1945,8 +1944,8 @@ try
         }
 
         ## It requires 'New-PSSession' to work with implicit credential to allow proxied command to create new session.
-        ## Implicit credential doesn't work in AppVeyor builder, so mark this test '-pending'.
-        It "Should have a new session when the disconnected session cannot be re-connected" -Pending {
+        ## Implicit credential doesn't work in the Azure DevOps builder, so mark this test '-pending'.
+        It "Should have a new session when the disconnected session cannot be re-connected" -Pending:(Test-IsVstsLinux -or Test-IsVstsWindows)
```
Comment (RDIL) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1]:
```suggestion
        It "Should have a new session when the disconnected session cannot be re-connected" -Pending:(Test-IsVstsLinux -or Test-IsVstsWindows) {
```

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (TravisEz13) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1]: Thanks

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1 [hunk 3]
```diff
@@ -1945,8 +1944,8 @@ try
         }
 
         ## It requires 'New-PSSession' to work with implicit credential to allow proxied command to create new session.
-        ## Implicit credential doesn't work in AppVeyor builder, so mark this test '-pending'.
-        It "Should have a new session when the disconnected session cannot be re-connected" -Pending {
+        ## Implicit credential doesn't work in the Azure DevOps builder, so mark this test '-pending'.
+        It "Should have a new session when the disconnected session cannot be re-connected" -Pending:(Test-IsVstsLinux -or Test-IsVstsWindows) {
```
Comment (TravisEz13) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1]:
```suggestion
        It "Should have a new session when the disconnected session cannot be re-connected" -Pending:(Test-IsVstsLinux -or Test-IsVstsWindows -or $IsMacOS) {
```

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1 [hunk 4]
```diff
@@ -1945,8 +1944,8 @@ try
         }
 
         ## It requires 'New-PSSession' to work with implicit credential to allow proxied command to create new session.
-        ## Implicit credential doesn't work in AppVeyor builder, so mark this test '-pending'.
-        It "Should have a new session when the disconnected session cannot be re-connected" -Pending {
+        ## Implicit credential doesn't work in the Azure DevOps builder, so mark this test '-pending'.
```
Comment (TravisEz13) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1]:
```suggestion
        ## Implicit credential doesn't work in the Windows Azure DevOps builder, so mark this test '-pending'.
        ## Also, this feature doesn't work on macOS or Linux
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@RDIL 	Don't worry about the CLA bot working on every commit.  It worked on the first commit `f1fb8b2`.  You can see by clicking the status indicator next to it.  Mentioning which commit it worked in, might be helpful though.

**Pull request merged**